src/main/scala/nova/scala/wrapper/FunctionalWrapper.scala

```diff
@@ -26,191 +26,242 @@ import nova.core.event.bus.EventListener
 
 /**
  * Implicitly converts Scala methods to Java methods.
+ *
+ * @deprecated Since Scala 2.12, any method which accepts a valid Java 8 functional interface
+ * can have a Scala lambda passed to it directly and all Scala FunctionN traits
+ * are now valid Java 8 functional interfaces, making this class unnecessary boilerplate.
  * @author anti344
  */
+@Deprecated
```

Comment:
Is there a reason we shouldn't delete this? We don't need backwards compat because we have no releases.